BankManagment/UnitOfWork/Entity/PaymentMethod.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BankManagmentDB.Entity
+{
+    public class PaymentMethod
```

Comment:
Models for Payment Method and Account Type contains the same fields, instead create a base class which has the properties and create two child class out of it.